Makefile

```diff
@@ -1,7 +1,13 @@
-.PHONY: vendor docs
+.PHONY: vendor docs build
 
 PACKAGES = $(shell go list ./... | grep -v /vendor/)
 
+ifneq ($(shell uname), Darwin)
+	EXTLDFLAGS = -extldflags "-static" #
```

Comment:
Some IDE, editors and tools may remove whitespace in the end. It is one of the solution how to keep spaces.